README.md

```diff
@@ -656,7 +656,8 @@ Here are some approaches to building applications with Snabbdom.
 * [Cycle.js](https://cycle.js.org/) –
   "A functional and reactive JavaScript framework for cleaner code"
   uses Snabbdom
-* [Vue.js](http://vuejs.org/) use a fork of snabbdom.
+* [Vue.js](http://vuejs.org/) uses a fork of snabbdom.
+* [stencil](https://stenciljs.com/) uses a [fork](https://github.com/ionic-team/stencil/blob/6193e74c9cca2ed7038c7e5fdf1fd925f8eba831/src/core/renderer/patch.ts) of snabbdom.
```

Comment:
I don't think we need to link to the file, just specified that this is a fork.

Reply:
I will love to see why do they need to fork it, if anyone has more info about it, and see if we can provide the APIs/features that they need to avoid the fork.